db/seeds.rb

```diff
@@ -6,9 +6,7 @@
 
 organization = Organization.create_or_find_by(name: '100eyes',
                                               upgrade_discount: 10,
-                                              business_plan: BusinessPlan.where.not(
-                                                valid_from: nil, valid_until: nil
-                                              ).first)
+                                              business_plan: BusinessPlan.first)
```

Comment:
Is this related?

Reply:
No, it's code creep.
Been removed in 4d4b66e2